apps/filebrowser/src/filebrowser/conf.py

```diff
@@ -40,6 +40,18 @@
   default=None,
   type=str)
 
+FILE_UPLOAD_CHUNK_SIZE = Config(
+  key="file_upload_chunk_size",
+  default=5000000,
```

Comment:
What is this value? 5MB?

Reply:
Yes its 5MB

Reply:
Any specific reason to reduce it that much? How was this calculated.
A bit worried it will affect Ozone upload then? cc: @agl29

Reply:
This is the size of an individual chunk the browser creates while uploading a file right? If so I don't think 5 MB is small for a chunk. @Harshg999 When you say it was reduced, are you referring to the upload_chunk_size for hadoop?

Reply:
Ahh I see, interpreted this config wrong. Thanks for clarifying @bjornalm! If this is for the file chunks at the browser side, then 5 MB looks good!

As a nit, @athithyaaselvam would it make sense to improve the config name and help message to incorporate that this chunk value is related to UI side file chunking?

Reply:
And just to be sure, is the file uploader expecting 5000000 or 5*1024*1024 when we say 5MB?

Reply:
Yes, this is the size of an individual chunk the browser creates while uploading a file. Changed the default value for 5MB from 5000000 to 5242880.